app/frontend/components/domains/permit-application/errors-box.tsx

```diff
@@ -22,20 +22,22 @@ export const ErrorsBox = ({ errorBox }: IErrorBoxProps) => {
       borderColor={"semantic.error"}
       borderWidth={"1px"}
       borderRadius="lg"
-      maxH="360px"
+      minH="85px"
+      maxH="calc(100vh - 360px)"
```

Comment:
what is the significance of 360px? should/could this be a variable?

Reply:
later will refactor to be a calculation .. 360px is the total height that the blue permit header + floating buttons take up on the screen. This is so that the error alert box doesn't collide / cover those.